### PR TITLE
Change the SRC_URI for AudioManager

### PR DESCRIPTION
**Pull request by DunderRoffe**
Change the SRC_URI for AudioManager from "git.projects.genivi.org" to
"github.com".

Add SRCREV since branch tracking is dangerous and can not be used offline.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb [hunk 1]
```diff
@@ -9,8 +9,9 @@ DEPENDS = "qtbase qtdeclarative qtwebkit"
 BRANCH = "Intreprid_stable_branch"
 VERSION = "6.1"
```
Comment (chbae) [inline meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb]: If you use SRCREV, you don't need to BRANCH and VERSION variables. Please remove those.

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb [hunk 2]
```diff
@@ -9,8 +9,9 @@ DEPENDS = "qtbase qtdeclarative qtwebkit"
...
 
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
 SRC_URI = "\
-    git://git.projects.genivi.org/AudioManager.git;branch=${BRANCH};tag=${VERSION};protocol=http \
+    git://github.com.genivi/AudioManager.git;protocol=http \
```
Comment (chbae) [inline meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb]: git://github.com/GENIVI/AudioManager.git;protocol=http

**Comment by chbae** [comment]
@DunderRoffe 

I agree with your scheme but you need to change somethings.

**Comment by DunderRoffe** [comment]
Good catch :) Now it is fixed.

**Review by zeenix — CHANGES_REQUESTED** [review]
"Change the SRC_URI for AudioManager"

Is not specific enough to be very helpful, "Change" being a very vague term. Let's be a bit more specific about the change.

Also you want to add "audiomanager: " prefix to commit (and hence "for AudioManager" becomes redundant).

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb [hunk 3]
```diff
@@ -6,11 +6,9 @@ LIC_FILES_CHKSUM = "file://AudioManagerPoC/README;md5=be0fe05c89ff7a2632eb080f72
 
 DEPENDS = "qtbase qtdeclarative qtwebkit"
 
-BRANCH = "Intreprid_stable_branch"
-VERSION = "6.1"
-
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
```
Comment (zeenix) [inline meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:9]: One more thing: This is a separate change, hence should be in a separate commit.

**Review by zeenix — CHANGES_REQUESTED** [review]
"Redefine the SRC_URI for AudioManager from "git.projects.genivi.org" to "github.com"."

This is not bad summary at all so this is just an improvement idea: I tend to try and avoid mentioning implementation details like "changing of SRC_URI" in commit log. So in here I'd have wrote:

"Fetch AudioManager from github.com instead of git.projects.genivi.org because [reasons]".

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb (hunk 3 above)
Comment (DunderRoffe) [reply meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:9]: Then I would have to rewrite the other part making an unnecessary step adding the branch stuff to the other URI. I get the point of keeping separate commits but this seems more weird than helpful.

Comment (zeenix) [reply meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:9]: > Then I would have to rewrite the other part 

Yes, that's true but now this will also force you to get into always thinking about dividing your changes from the very beginning and plan better.

> I get the point of keeping separate commits but this seems more weird than helpful.

You either get it or you don't. :) We need to have same rules for everyone and every commit so in this case it might not be very helpful to divide but once you start bending the rules, where do you draw the line?

Comment (zeenix) [reply meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:9]: Oh and summary/shortlog can only describe one change so if you have any other changes (no matter how small) in your commit, they'll be missed when someone is reading through shortlog of changes.

**Review by chbae — APPROVED** [review]

File: meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb (hunk 1 above)
Comment (zeenix) [reply meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb]: But he is no longer using SRCREV (yet).

**Review by zeenix — APPROVED** [review]
